cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1020,19 +1028,398 @@ subroutine input_data
+            write(nu_diag,1007) ' e_ratio          = ', e_ratio, ' aspect ratio of ellipse'
+         elseif (kdyn == 2) then
+            tmpstr2 = ' elastic-anisotropic-plastic dynamics'
+         elseif (kdyn < 1) then
+            tmpstr2 = ' dynamics disabled'
+         endif
+         write(nu_diag,1022) ' kdyn             = ', kdyn,trim(tmpstr2)
+         if (kdyn >= 1) then
+            if (revised_evp) then
+               tmpstr2 = ' revised EVP formulation used'
+            else
+               tmpstr2 = ' revised EVP formulation not used'
+            endif
+            write(nu_diag,1012) ' revised_evp      = ', revised_evp,trim(tmpstr2)
+            write(nu_diag,1022) ' kevp_kernel      = ', kevp_kernel,' EVP solver'
+
+            write(nu_diag,1022) ' ndtd             = ', ndtd, ' number of dynamics/advection/ridging/steps per thermo timestep'
+            write(nu_diag,1022) ' ndte             = ', ndte, ' number of EVP or EAP subcycles'
+            write(nu_diag,1007) ' arlx             = ', arlx, ' stress equation factor alpha'
+            write(nu_diag,1007) ' brlx             = ', brlx, ' stress equation factor beta'
+
+            if (trim(coriolis) == 'latitude') then
+               tmpstr2 = ': latitude-dependent Coriolis parameter'
+            elseif (trim(coriolis) == 'contant') then
+               tmpstr2 = ' = 1.46e-4/s'
+            elseif (trim(coriolis) == 'zero') then
+               tmpstr2 = ' = 0.0'
+            endif
+            write(nu_diag,*) 'coriolis         = ',trim(coriolis),trim(tmpstr2)
+
+            if (ktransport == 1) then
+               tmpstr2 = ' transport enabled'
+               if (trim(advection) == 'remap') then
+                  tmpstr2 = ': linear remapping advection'
+               elseif (trim(advection) == 'upwind') then
+                  tmpstr2 = ': donor cell (upwind) advection'
+               endif
+               write(nu_diag,*) 'advection        = ', trim(advection),trim(tmpstr2)
+            else
+               tmpstr2 = ' transport disabled'
+            endif
+            write(nu_diag,1022) ' ktransport       = ', ktransport,trim(tmpstr2)
+
+            if (basalstress) then
+               tmpstr2 = ' use basal stress parameterization for landfast ice'
+            else
+               tmpstr2 = ' basal stress not used for landfast ice'
+            endif
+            write(nu_diag,1012) ' basalstress      = ', basalstress,trim(tmpstr2)
+            if (basalstress) then
+               write(nu_diag,1007) ' k1               = ', k1, ' free parameter for landfast ice'
+               write(nu_diag,1007) ' k2               = ', k2, ' free parameter for landfast ice'
+               write(nu_diag,1007) ' alphab           = ', alphab, ' factor for landfast ice'
+               write(nu_diag,1007) ' threshold_hw     = ', threshold_hw, ' max water depth for grounding ice'
+               write(nu_diag,1007) ' Ktens            = ', Ktens, ' tensile strength factor'
```

Comment:
@eclare108213, I think this was overlooked but `Ktens` should not be inside the `if (basalstress)` block...

`Ktens` can be used even if the seabed stress parameterization is not used...

@JFLemieux73